admin/templates/preprints/provider.html

```diff
@@ -9,9 +9,9 @@
             </a>
             <div class="collapse" id="collapseChangeProvider">
                 <div class="well">
-                    <form action="{% url 'preprints:preprint' guid=preprint.guid %}" method="post">
+                    <form action="{% url 'preprints:preprint-provider' guid=preprint.guid %}" method="post">
                         {% csrf_token %}
-                        {{ form.as_p }}
```

Comment:
Once there were multiple named forms this view had to change, I would not embrace a mono-form system.